helm/exporter-node/templates/daemonset.yaml

```diff
@@ -21,37 +21,33 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: "{{ .Values.image.pullPolicy }}"
           args:
-          {{ if lt .Values.image.tag "v0.15.0" }}
-            - --collector.procfs=/host/proc
-            - --collector.sysfs=/host/sys
-          {{ else }}
-            - --path.procfs=/host/proc
-            - --path.sysfs=/host/sys
-          {{ end }}
-            - --web.listen-address=0.0.0.0:{{ .Values.service.containerPort }}
+          - --web.listen-address=0.0.0.0:{{ .Values.service.containerPort }}
+          {{- if .Values.container }}
```

Comment:
Please use one line if https://stackoverflow.com/a/43026557

Reply:
the issue with the inline if is that both the operands of the and seem to be evaluated. 
when i set the ```container``` key to ```null``` (let's say someone wants the most basic exporter config possible), i get ```Error: render error executing exporter-node/templates/daemonset.yaml:25:46 at <.Values.container.ar...>: can't evaluate field args in type interface {}```

Reply:
What about we assume Values.container.args should not be null?

Reply:
ok, the following would work to disable the default args:
```
container: 
  args: []
```

item for volumes & mounts

Reply:
There’s no document for now, I’ll write docs as soon I move this chart to the K8s charts upstream